api/envoy/type/matcher/v3/string.proto

```diff
@@ -53,6 +53,11 @@ message StringMatcher {
     // The input string must match the regular expression specified here.
     RegexMatcher safe_regex = 5 [(validate.rules).message = {required: true}];
   }
+
+  // If true, indicates the exact/prefix/suffix matching should be case insensitive. This has no
+  // effect for the safe_regex match.
+  // For example, the matcher *data* will match both input string *Data* and *data* if set to true.
+  bool ignore_case = 6;
```

Comment:
Fly-by comment: should this be v4alpha?

Reply:
No, changes to v3 is allowed without breaking compatibility. And we're still in the grace period allowing changing v2.